CHANGES.rst

```diff
@@ -4,7 +4,9 @@ Histórico de Alterações
 1.5rc1 (unreleased)
 ^^^^^^^^^^^^^^^^^^^
 
-- Nothing changed yet.
+- Adiciona Products.RedirectionTool ao IDG;
```

Comment:
Acho que seria interessante uma frase de uma linha informando o que essa dependência faz, assim, quando alguém for ler o changelog, entende a motivação.

Reply:
a frase está na linha seguinte; se tiver outra sugestão por favor comentar ai.